content/soarian/r4/clinical/diagnostics/diagnostic-report.md

```diff
@@ -0,0 +1,145 @@
+---
+title: DiagnosticReport | R4 API
+---
+
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Overview
+
+The DiagnosticReport resource includes information that is typically provided by a diagnostic service when investigations are complete. The information includes laboratory, microbiology, results reporting, and clinical notes exchange. Clinical notes include the three categories of Cardiology, Pathology, and Radiology.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
+The search results include the following fields if they contain values:
+
```

Comment:
How about the DiagnosticReport.identifier field?

Reply:
We do not support DrxReport.identifier  field, we have identifier code and abbreviation in code section
User Story:
![image](https://user-images.githubusercontent.com/37074481/138032210-79245e25-9d26-4cf7-a7b2-c34676545078.png)
Documentation Word file:
![image](https://user-images.githubusercontent.com/37074481/138032282-e7d83cac-21d9-4484-a794-34d53ee69c2d.png)